docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
 # Data distribution
 
-When you create a table, you can specify the data distribution method by configuring the partitioning and bucketing strategy in the table. An appropriate data distribution method helps evenly distribute data across the nodes of your StarRocks cluster. It can also reduce the data scanned and make full use of the concurrency of the cluster during queries, thereby improving query performance.
+Configuring appropriate partitioning and bucketing at table creation data can help to achieve even data distribution and improve query performance. Even data distribution means dividing the data into subsets according to certain rules and distributing them evenly across different nodes. It can also reduce the data scanned and make full use of the concurrency of the cluster during queries, thereby improving query performance.
 
-> **NOTICE**
+> **NOTE**
 >
-> - Since v3.1, you do not need to specify the bucketing key when creating a table or adding partitions. StarRocks supports random bucketing, which randomly distributes data across all buckets. For more information, see [Random bucketing](#random-bucketing-since-v31).
+> - Since v3.1, you can not specify the bucketing key in the DISTRIBUTED BY clause when creating a table or adding a partition. StarRocks supports random bucketing, which randomly distributes data across all buckets. For more information, see [Random bucketing](#random-bucketing-since-v31).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/dc9d3cc6-5068-43ea-94fe-9a0b1ff5b08c)
First, in technical docs suggest using "cannot" instead of "can not" or "can't". Second, "cannot" here convey a totally different meaning compared with "do not need to". See the Chinese version, which clearly states that "you do not need to" instead of "cannot".